kapitan/targets.py

```diff
@@ -86,7 +84,7 @@ def compile_targets(
                 logger.info("No changes since last compilation.")
                 return
 
-    pool = multiprocessing.Pool(parallel)
+    pool = multiprocessing.get_context("spawn").Pool(parallel)
```

Comment:
It's better to change it here: https://github.com/kapicorp/kapitan/blob/1c1a00b5266e9cc57c95e4430240f325af9ec088/kapitan/cli.py#L102

Reply:
Thanks @pvanderlinden - updated as requested